libraries/standard/mqtt/cbmc/proofs/MQTT_SerializeConnect/Makefile

```diff
@@ -27,7 +27,7 @@ INCLUDES +=
 
 # This function does not coincide with the call graph of MQTT_Serialize, but are
 # found by CBMC during processing in logs/MQTT_Connect_harness3.txt. We remove
-# the function body to increase the execution time of this proof.
+# the function body to help with coverage.
```

Comment:
the function body to improve coverage accuracy.

Reply:
thanks